frontend/src/components/profile/PageLayout.jsx

```diff
@@ -10,15 +10,34 @@ import EditButton from './EditButton';
 class PageLayout extends React.Component {
     constructor(props) {
         super(props);
-        this.state = { key: 'aboutMe' }
+        this.state = { key: 'aboutMe', items: [], isLoaded:false }
```

Comment:
This is not mapped from reducer and thus will only render once if in constructor. 

Are you sure it is working properly?